packages/contracts/contracts/core/IDAO.sol

```diff
@@ -31,72 +31,97 @@ abstract contract IDAO {
     /// @param _metadata The IPFS hash of the new metadata object
     function setMetadata(bytes calldata _metadata) external virtual;
 
+    /// @notice Emitted when the DAO metadata is updated
+    /// @param metadata The IPFS hash of the new metadata object
     event MetadataSet(bytes metadata);
 
     /// @notice If called, the list of provided actions will be executed.
-    /// @dev It run a loop through the array of acctions and execute one by one.
-    /// @dev If one acction fails, all will be reverted.
+    /// @dev It runs a loop through the array of actions and execute them one by one.
+    /// @dev If one action fails, all will be reverted.
     /// @param _actions The aray of actions
     function execute(uint256 callId, Action[] memory _actions)
         external
         virtual
         returns (bytes[] memory);
 
+    /// @notice Emitted when a proposal is executed
+    /// @param actor The address of the caller
+    /// @param callId The id of the call
+    /// @dev The value of callId is defined by the component/contract calling the execute function.
+    ///      A Component implementation can use it, for example, as a nonce.
+    /// @param actions Array of actions executed
+    /// @param execResults Array with the results of the executed actions
     event Executed(address indexed actor, uint256 callId, Action[] actions, bytes[] execResults);
 
     /// @notice Deposit ETH or any token to this contract with a reference string
     /// @dev Deposit ETH (token address == 0) or any token with a reference
     /// @param _token The address of the token and in case of ETH address(0)
     /// @param _amount The amount of tokens to deposit
-    /// @param _reference The deposit reference describing the reason of it
+    /// @param _reference The reference describing the deposit reason
     function deposit(
         address _token,
         uint256 _amount,
         string calldata _reference
     ) external payable virtual;
 
+    /// @notice Emitted when a deposit is made
+    /// @param sender The address of the sender
+    /// @param token The address of the token deposited
+    /// @param amount The amount of tokens deposited
```

Comment:
feels like an unfinished sentences
perhaps: `/// @param amount The amount of tokens deposited in the DAO`
or: `/// @param amount The amount of deposited tokens`

same for amount

Reply:
I will change it to `/// @notice Emitted when a deposit is made to the DAO`. This way, it does not need to be included twice in the `/// @param` comments